stdlib/public/core/Array.swift

```diff
@@ -1996,6 +1996,10 @@ extension Array where Element : Differentiable {
         selfElement.tangentVector(from: cotangentVectorElement)
       })
     }
+    
+    public var description: String {
+      return base.description
+    }
   }
 }
```

Comment:
Could you define this as an extension instead like the `Equatable` conformance below?

```swift
public extension Array.DifferentiableView : CustomStringConvertible {
  var description: String {
    ...
  }
}
```

Reply:
Oh okay sure!